README.md

```diff
@@ -35,20 +35,20 @@ To get more help on the Angular CLI use `ng help` or go check out the [Angular C
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.5 with additional steps to include Vaadin components. You can recreate the project by following along these steps:
 
-First [install npm](https://docs.npmjs.com/getting-started/installing-node)
+First [install yarn](https://yarnpkg.com/lang/en/docs/install)
 
 ``` bash
-$ npm install -g @angular/cli
+$ yarn install -g @angular/cli
```

Comment:
This isn't a valid command

Reply:
Two commits were not merged here, fixed now.